### PR TITLE
Support asterisk on list

### PR DESCRIPTION
**Pull request by nakulkar-msft**

### CODE REVIEW

File: cmd/list.go
Comment (gapra-msft) [inline cmd/list.go:90]: @nakulkar-msft  can we wait until adele's new e2e test framework is merged and add e2e tests for this and then support this feature?

Comment (siminsavani-msft) [reply cmd/list.go:90]: +1

**Review by siminsavani-msft — CHANGES_REQUESTED** [review]

File: cmd/list.go
Comment (siminsavani-msft) [reply cmd/list.go:90]: +1

**Pull request closed without merging**
